js/lib/urlutil.js

```diff
@@ -486,6 +486,14 @@ const UrlUtil = {
     return url
       .replace(/((#?\/?)|(\/#?))$/, '') // remove trailing # and /
       .trim() // remove whitespaces
+  },
+
+  shortenUrlToDomainTLD: (url) => {
+    if (url !== null) {
```

Comment:
`url !== null` let's change this to `url != null`. This way we are checking `null` and `undefined`